Marlin/src/gcode/feature/pause/M125.cpp

```diff
@@ -78,7 +78,13 @@ void GcodeSuite::M125() {
   // If possible, show an LCD prompt with the 'P' flag
   const bool show_lcd = TERN0(HAS_LCD_MENU, parser.boolval('P'));
 
-  TERN_(POWER_LOSS_RECOVERY, if (recovery.enabled) recovery.save(true));
+  const float raised_z = _MAX(park_point.z, _MIN(Z_MAX_POS, current_position.z
```

Comment:
`park_point.z` is not a position. It is just the Z raise value. So you don't want to use it in `G92.9 Z%s`. I will modify the code so that `recovery.resume` applies the value correctly.

Reply:
Yes I know, I "misused" it in order to avoid introducing another variable. 

Anyway, big thanks for helping with this commit!!

If necessary, I will check your adaptions on friday.